### PR TITLE
add new testcase

### PR DESCRIPTION
**Pull request by smitapawar0195**

### CODE REVIEW

File: src/test/java/com/test/TestSmita.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.test;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+public class TestSmita {
+	@Test
+	public void m2() {
```
Comment (Ajiket97) [inline src/test/java/com/test/TestSmita.java:8]: please check

**Review by Ajiket97 — COMMENTED** [review]
add other tesst cases also

File: src/test/java/com/test/TestSmita.java (hunk 1 above)
Comment (smitapawar0195) [reply src/test/java/com/test/TestSmita.java:8]: ok

**Comment by Ajiket97** [comment]
ok

**Pull request merged**